domain/src/main/java/edu/nextstep/camp/caculator/domain/Expression.kt

```diff
@@ -0,0 +1,29 @@
+package edu.nextstep.camp.caculator.domain
+
+class Expression (
+    private val expressionString: String) {
+    private val expressionList = mutableListOf<String>()
```

Comment:
기존 Calculator이 가지고 있던 표현식을 파싱하는 역할을 잘 분리해 주셨습니다. 👍
다만 포멧팅이 좀 헷갈리게 되어있네요. 두 프로퍼티 모두 생성자 파라미터 처럼 보였습니다. 

```suggestion
class Expression (private val expressionString: String) {
    private val expressionList = mutableListOf<String>()
```